### DISCUSSION TITLE
How to load dotenv files within a lane?

### DISCUSSION THREAD

**Discussion by Nonouf** [fastlane/fastlane#20138] [Q&A] [answered]
Hi,

I have a few environments setup and I'd like to create a lane which will call the same command several time with different environment each time without the use of `sh`.

For example, instead of doing :
```ruby
lane :captures_for_store do
    Dir.chdir("..") do
        sh("bundle exec fastlane snapshot --env envOne")
        sh("bundle exec fastlane snapshot --env envTwo")
        sh("bundle exec fastlane snapshot --env envThree")
    end
end
```
I'd like to do something like:
```ruby
lane :captures_for_store do
    snapshot(env: "envOne")
    snapshot(env: "envTwo")
    snapshot(env: "envThree")
end
```
Is there any way to load an env within a Fastfile?

Thanks!

**Comment by lucgrabowski** [comment, marked as answer]
You could use `dotenv` gem directly in your lane's code https://github.com/bkeepers/dotenv
Fastlane is using `dotenv` in https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/helper/dotenv_helper.rb and you can take a look how it's done there.
